### PR TITLE
web/settings/audio: disable bitrate section when not applicable

### PR DESCRIPTION
**Pull request by halozat**
please delete my closed pr thanks :3

if the audio format is either `best` or `wav`,
the bitrate selector should be greyed out, 
cause it's lossless, or not necessarily re-encoded.
format: best (not re-encoded)
format: wav (lossless)
format mp3 (lossy)

### CODE REVIEW

File: web/i18n/en/settings.json
Comment (wukko) [inline web/i18n/en/settings.json:1]: why

**Review by wukko — CHANGES_REQUESTED** [review]

File: web/i18n/en/settings.json
Comment (wukko) [reply web/i18n/en/settings.json:1]: we're a 4 space community

Comment (halozat) [reply web/i18n/en/settings.json:1]: oh shit sorry my prettier was left on by accident. 😭

Comment (halozat) [reply web/i18n/en/settings.json:1]: i had made a prettier config that is 98% accurate to the cobalt codebase btw.
that 2% is made up of unconsistency, line wrapping

File: web/src/routes/settings/audio/+page.svelte [hunk 1]
```diff
@@ -7,6 +7,7 @@
     import Switcher from "$components/buttons/Switcher.svelte";
     import SettingsButton from "$components/buttons/SettingsButton.svelte";
     import SettingsToggle from "$components/buttons/SettingsToggle.svelte";
+    import settings from "$lib/state/settings";
```
Comment (wukko) [inline web/src/routes/settings/audio/+page.svelte:10]: this import should be on very top, before translations

**Review by wukko — CHANGES_REQUESTED** [review]

File: web/i18n/en/settings.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
 
     "audio.bitrate": "audio bitrate",
     "audio.bitrate.kbps": "kb/s",
-    "audio.bitrate.description": "bitrate applies only to audio conversion. cobalt can't improve the source audio quality, so choosing a bitrate over 128kbps may inflate the file size with no audible difference. perceived quality may vary by format.",
+    "audio.bitrate.description": "bitrate only applies to audio conversion into lossy audio formats. cobalt can't improve the source audio quality, so choosing a bitrate over 128kbps may inflate the file size with no audible difference. perceived quality may vary by format.",
```
Comment (wukko) [inline web/i18n/en/settings.json:53]: bitrate is applied only when converting audio to a lossy format.

**Review by wukko — CHANGES_REQUESTED** [review]

File: web/i18n/en/settings.json [hunk 3]
```diff
@@ -50,7 +50,7 @@
 
     "audio.bitrate": "audio bitrate",
     "audio.bitrate.kbps": "kb/s",
-    "audio.bitrate.description": "bitrate applies only to audio conversion. cobalt can't improve the source audio quality, so choosing a bitrate over 128kbps may inflate the file size with no audible difference. perceived quality may vary by format.",
+    "audio.bitrate.description": "bitrate is only applied upon converting audio to a lossy format. cobalt can't improve the source audio quality, so choosing a bitrate over 128kbps may inflate the file size with no audible difference. perceived quality may vary by format.",
```
Comment (wukko) [inline web/i18n/en/settings.json:53]: "bitrate is **applied only** upon converting..."

**Review by wukko — CHANGES_REQUESTED** [review]

File: web/i18n/en/settings.json [hunk 4]
```diff
@@ -50,7 +50,7 @@
 
     "audio.bitrate": "audio bitrate",
     "audio.bitrate.kbps": "kb/s",
-    "audio.bitrate.description": "bitrate applies only to audio conversion. cobalt can't improve the source audio quality, so choosing a bitrate over 128kbps may inflate the file size with no audible difference. perceived quality may vary by format.",
+    "audio.bitrate.description": "bitrate is applied only upon converting audio to a lossy format. cobalt can't improve the source audio quality, so choosing a bitrate over 128kbps may inflate the file size with no audible difference. perceived quality may vary by format.",
```
Comment (wukko) [inline web/i18n/en/settings.json:53]: bitrate is applied only **_when_** converting...

**Review by wukko — APPROVED** [review]

**Pull request merged**